earth_enterprise/src/fusion/autoingest/sysman/AssetHandleD.h

```diff
@@ -415,7 +327,7 @@ class MutableDerivedAssetHandleD_ : public DerivedBase_, public MutableBase_
  public:
```

Comment:
For some reason it won't let me comment on line 411/323.  I _think_ that line is doing a little extra work since this->ref is a std::string and dirtyMap is SharedString now.   We can probably also replace `insert(std::make_pair` with `dirtyMap.emplace` since we have c++11 and save a little more

Reply:
I haven't followed the inheritance chain back, but I assume that this inherits from AssetHandle which is where `ref` is defined. But that's been changed to a `SharedString` (unless I'm missing something), so I think there's no extra conversion there. But good point about `emplace`. I can switch he insert calls to that.

Reply:
I think you're right.  I followed back to a different ref the first time which was a std::string but I think the one in AssetHandle is the correct one.  I'll leave the emplace up to you.  Probably many places could benefit from that now that it is available.  I won't be too offended if you decide to draw the line where things stand